test/test-mda-models/aissemble-test-data-delivery-pyspark-model-basic/pom.xml

```diff
@@ -87,38 +87,16 @@
                     </execution>
                 </executions>
             </plugin>
-            <!--NB: inactive by default until https://github.com/boozallen/aissemble/issues/4 is resolved: -->
             <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>exec-maven-plugin</artifactId>
-                <version>3.1.0</version>
-                <executions>
-                    <execution>
-                        <id>test-chart</id>
-                        <phase>compile</phase>
-                        <goals>
-                            <goal>exec</goal>
-                        </goals>
-                        <configuration>
-                            <executable>helm</executable>
-                            <arguments>
-                                <argument>template</argument>
-                                <argument>aissemble-spark-application-chart</argument>
-                                <argument>--repo</argument>
-                                <argument>${aissemble.helm.repo}</argument>
-                                <argument>--version</argument>
-                                <argument>${version.aissemble}</argument>
-                                <argument>--values</argument>
-                                <argument>
-                                    src/aissemble_test_data_delivery_pyspark_model_basic/resources/apps/pyspark-data-delivery-basic-base-values.yaml,tests/resources/apps/pyspark-data-delivery-basic-test-values.yaml
-                                </argument>
-                                <argument>-s</argument>
-                                <argument>templates/deployment.yaml</argument>
-                            </arguments>
-                            <outputFile>target/apps/pyspark-data-delivery-basic-test-chart.yaml</outputFile>
-                        </configuration>
-                    </execution>
-                </executions>
+                <groupId>${group.helm.plugin}</groupId>
+                <artifactId>helm-maven-plugin</artifactId>
```

Comment:
S: The original plugin setup was based on what we generate for downstream projects.  I like this solution better, but if we don't align it with the expected format for downstream projects, then we aren't doing a great job of verifying the behavior.

Reply:
Yes upon investigating helm-maven-plugin doesn't support  any config option to output specific format that we want. They have output directory to dump files but they don't have option to rename to what we want.

Unfortunately I am going to fall back to using exec-maven-plugin as it provides outputFile.

Great callout! definitely missed downstream project use case.